pkg/pluginmanager/manager.go

```diff
@@ -997,13 +993,13 @@ func InstallPluginsFromLocalSource(pluginName, version string, target configtype
 	}
 
 	if len(matchedPlugins) == 1 {
-		return installOrUpgradePlugin(&matchedPlugins[0], FindVersion(matchedPlugins[0].RecommendedVersion, version), installTestPlugin)
+		return installOrUpgradePlugin(&matchedPlugins[0], matchedPlugins[0].RecommendedVersion, installTestPlugin)
```

Comment:
Ok, I understand.  So moving `FindVersion()` is good, but you need to leave `InstallPluginsFromLocalSource()` like before where it calls `installOrUpgradePlugin()` passing it `version` instead of `matchedPlugins[i].RecommendedVersion`

Reply:
My bad. Got it. I misunderstood earlier.